test/worker/test-worker.ts

```diff
@@ -0,0 +1,41 @@
+import test from "ava";
+import * as sinon from "sinon";
+import { WorkerWrapperImpl } from "@worker/worker";
+import Storage from "@worker/storage";
+import { StoredWordList } from "@worker/storage";
+import * as compiler from "@predictive-text-studio/lexical-model-compiler";
+
+test("compile model should throws error when no file is found in the IndexedDB", async (t) => {
+  const storageStub = new Storage();
+  sinon.stub(storageStub, "fetchAllFiles").returns(Promise.resolve([]));
+
+  const workerWrapper = new WorkerWrapperImpl(storageStub);
+
+  const error = await t.throwsAsync(workerWrapper.compileModel());
+  t.is(error.message, "Cannot find any file in the IndexedDB");
```

Comment:
I'm hesitant to test the _exact_ string of an error message like this, but... maybe it doesn't matter here because the error message is unlikely to change? Maybe?